cmd/root/io.go

```diff
@@ -21,7 +21,8 @@ func initOutputFlag(cmd *cobra.Command) *outputFlag {
 	// Configure defaults from environment, if applicable.
 	// If the provided value is invalid it is ignored.
 	if v, ok := env.Lookup(cmd.Context(), envOutputFormat); ok {
-		f.output.Set(v)
+		// QQQ log the error?
+		_ = f.output.Set(v)
```

Comment:
we might even panic here as it's unexpected error to not being able to set it

Reply:
To be safe here as well, I'm leaving this unchanged and just annotating it with nolint